picamera2/sensor_format.py

```diff
@@ -0,0 +1,19 @@
+import re
+
+
+class SensorFormat():
+    def __init__(self, fmt_string):
+        pixels, self.packing = fmt_string.split("_")
+        self.bit_depth = int(re.search("\d+$", pixels).group())
+        self.arrangement = re.search("[RGB]+", pixels).group()
```

Comment:
All good here, but I find "arrangement" slightly vague... maybe "bayer_order" would be more specific?